helm/templates/fluent-bit-daemonset.yaml

```diff
@@ -28,7 +31,7 @@ spec:
         - name: AWS_REGION
           value: {{ .Values.region }}
         - name: CLUSTER_NAME
-          value: {{ .Values.clusterName | quote }}
+          value: {{ include "kubernetes-cluster.name" . }}
```

Comment:
@movence doesnt this bring back the bug you fixed for numeric only cluster names not being quoted?